integration_tests/src/main/python/misc_test.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from data_gen import *
+from spark_session import with_gpu_session
+
+import uuid
+
+
+# GPU uuid function does not guarantee the result is the same as CPU,
+# only guarantee it produces unique UUIDs
+def test_uuid():
+    def gen_uuids(spark):
+        return unary_op_df(spark, int_gen, length=num_rows).selectExpr(["a", "uuid() as uuid"]).collect()
```

Comment:
Not sure how it works in Python, since `num_rows` is used here before being defined at line #35.

Reply:
It works:
```python
>>> def f1():
...     return 5 + a
... 
>>> a = 5
>>> f1()
10
```